teps/0069-support-retries-for-custom-task-in-a-pipeline.md

```diff
@@ -219,8 +219,4 @@ Not there yet!
 
 ## References (optional)
 
-<!--
-Use this section to add links to GitHub issues, other TEPs, design docs in Tekton
-shared drive, examples, etc. This is useful to refer back to any other related links
-to get more details.
--->
+* [tektoncd/pipeline PR #4327 - TEP-69 implemented](https://github.com/tektoncd/pipeline/pull/4327)
```

Comment:
Hey thanks @abayer !!

There is a section in TEP: `## Implementation Pull request(s)` You might want to update the PR reference there.

Reply:
Whoops, missed that - doing so now.

Reply:
Updated!